README.md

```diff
@@ -1,216 +1,75 @@
 # Anvil
 
+https://hub.docker.com/repository/docker/saicoss/anvil
```

Comment:
I personally like this as a hyperlink, but if thats not part of my style, ignore this comment

Reply:
Let's leave it for right now. If I change it it will invalidate the approvals when i push a new commit. I'll add an issue to fix it later.